README.md

```diff
@@ -1,43 +1,8 @@
-<a name="readme-top"></a>
-
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
-OPTIONAL SECTIONS:
-- FAQ
-
-After you're finished please remove all the comments and instructions!
--->
-
-<div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
-  <br/>
-
-  <h3><b>Microverse README Template</b></h3>
-
-</div>
-
-<!-- TABLE OF CONTENTS -->
+<h1><b>Hello Microverse</b></h1>
 
 # 📗 Table of Contents
 
+-
 - [📖 About the Project](#about-project)
   - [🛠 Built With](#built-with)
```

Comment:
- Good job using the required Microverse readme template, but in order to make your readme file more professional, please remove the `Live Demo` section header on line `10` of your readme file, since this page is not deployed yet 👍 👍